src/routes/index.svelte

```diff
@@ -104,6 +140,25 @@
         }
     }
 
+    const saveSymbols = history => {
+        const currentSymbols = Array.from(history.keys())
+        let symbolList = ''
+
+        if (currentSymbols.length > 0) {
+            symbolList = currentSymbols.join(',')
+            localStorage.setItem(LS_SYMBOLS, symbolList)
```

Comment:
In another PR, might be worth creating a localStorage.utils.js with the following:

`const setLocalStorageItem = (key, value) => localStorage.setItem(key, value);

`const getLocalStorageItem = (key, value) => localStorage.getItem(key, value);`

So we can do this:
`setLocalStorageItem(LS_SYMBOLS, symbolList)`

Reply:
Sure 👍